main/acle.md

```diff
@@ -2756,9 +2757,7 @@ The following table lists the architectures feature mapping for AArch64
    | 490           | `FEAT_SSBS`              | ssbs          | ```ID_AA64PFR1_EL1.SSBS >= 0b0001```      |
    | 500           | `FEAT_SSBS2`             | ssbs2         | ```ID_AA64PFR1_EL1.SSBS >= 0b0010```      |
    | 510           | `FEAT_BTI`               | bti           | ```ID_AA64PFR1_EL1.BT >= 0b0001```        |
-   | 520           | `FEAT_LS64`              | ls64          | ```ID_AA64ISAR1_EL1.LS64 >= 0b0001```     |
-   | 530           | `FEAT_LS64_V`            | ls64_v        | ```ID_AA64ISAR1_EL1.LS64 >= 0b0010```     |
-   | 540           | `FEAT_LS64_ACCDATA`      | ls64_accdata  | ```ID_AA64ISAR1_EL1.LS64 >= 0b0011```     |
+   | 520           | `FEAT_LS64`,`FEAT_LS64_V`,<br>`FEAT_LS64_ACCDATA`| ls64 | ```ID_AA64ISAR1_EL1.LS64 >= 0b0011``` |
```

Comment:
```suggestion
   | 520           | `FEAT_LS64_ACCDATA`| ls64 | ```ID_AA64ISAR1_EL1.LS64 >= 0b0011``` |
```
I don't think there should be multiple architecture features listed. It makes it ambiguous how you test if the feature is present or not.

Reply:
That's the whole point I am trying to make, I am suggesting that the FMV feature named `ls64` means all of {FEAT_LS64,FEAT_LS64_V,FEAT_LS64_ACCDATA}. It is better to make it explicit in the specification rather than letting the user figure out the architecture dependency FEAT_LS64 -> FEAT_LS64_V ->FEAT_LS64_ACCDATA themselves in my opinion.

Reply:
Relevant features for others items are listed too in the table already.
Here this is a favour for the reader how looks for `FEAT_LS64` to be sure the `ls64` is the right option to choose without looking up the bits in the `ID_AA64ISAR1_EL1`

Reply:
As far as I understand we don't want to list architectural features that are implied if they have a corresponding FMV feature in place.

Reply:
Ok fair enough